### DISCUSSION TITLE
Obtaining sorted data coordinates from slices

### DISCUSSION THREAD

**Discussion by YsfDemiroglu** [pyvista/pyvista#2464] [Q&A] [answered]
Hi,
I am using `mesh.slice(normal=, origin=)` for slicing a surface mesh. However, `coordinates = slice.points` returns almost randomly sorted data coordinates. Since I need to perform line integration over the closed curves, I need to take these points in order. I have a similar issue in pvpython but I found a different filter which provides sorted coordinates:
`PV.PlotOnIntersectionCurves(registrationName='PlotOnIntersectionCurves1', Input=vtkFile)`

I wanted to ask whether this type of command exists in pyvista to get sorted coordinates of slices. 

Thanks.

**Comment by whophil** [comment, marked as answer]
This appears to be the same question as #2798 , for which I have posted a solution.

**Reply by YsfDemiroglu** [reply]
Thank you very much.
